packages/framework/src/Model/Product/Accessory/ProductAccessoryFacade.php

```diff
@@ -23,9 +23,9 @@ public function __construct(protected readonly ProductAccessoryRepository $produ
      * @param int|null $limit
      * @return \Shopsys\FrameworkBundle\Model\Product\Product[]
      */
-    public function getTopOfferedAccessories(Product $product, $domainId, PricingGroup $pricingGroup, $limit)
+    public function getOfferedAccessories(Product $product, int $domainId, PricingGroup $pricingGroup, int $limit = null)
```

Comment:
if the default value is null, I think we should have the type nullable, otherwise it's confusing. 

_phpstan does not complain about invalid type in method call due to different annotation from the type. I suggest keeping annotations and types in sync_